Gemfile.lock

```diff
@@ -356,17 +352,13 @@ GEM
     ruby-progressbar (1.9.0)
     ruby_dep (1.5.0)
     safe_yaml (1.0.4)
-    sass (3.5.2)
-      sass-listen (~> 4.0.0)
-    sass-listen (4.0.0)
-      rb-fsevent (~> 0.9, >= 0.9.4)
-      rb-inotify (~> 0.9, >= 0.9.7)
-    sass-rails (5.0.3)
-      railties (>= 4.0.0, < 5.0)
+    sass (3.4.25)
+    sass-rails (5.0.7)
```

Comment:
Shouldn't these change be pinned in the Gemfile?

Reply:
See my comment on `sass`.